### ISSUE TITLE
advantage of converting the Darknet-model to a Keras-model

### ISSUE THREAD

**Issue by phamngocthanhtrung** [Adamdad/keras-YOLOv3-mobilenet#13]
Hi, 
what is the advantage of converting the Darknet-model to a Keras-model ?
Best Regards

**Comment by Adamdad** [comment]
the structure is the same, but keras is easier to understand and wide-used

**Comment by phamngocthanhtrung** [comment]
> the structure is the same, but keras is easier to understand and wide-used

yes thank you, and the runtime stays equal?

**Comment by Adamdad** [comment]
No, because darknet is based on pure c, so darknet is faster, but hard to comprehend. Darknet is faster than tensorflow and keras.

**Comment by Adamdad** [comment]
That‘s not always true. Different Deep learning Framework use different tricks and skills and on their implementation. For example, the parameter initialization is very much important for the precision of CNN model, with different frameworks using different strategies.
